helpers/schemas/location.js

```diff
@@ -111,6 +112,20 @@ const address = joi.string().invalid('').required().error(errors => {
   return errors;
 });
 
+const place_id = joi.string().allow('').required().error(errors => {
+  errors.forEach(err => {
+    switch (err.type) {
+      case 'string.base':
+        err.message = 'place_id must be a string';
+      case 'any.required':
+        err.message = 'place_id key is required, but value can be empty';
```

Comment:
err.message seems to be long 
```err.message ='place_id field is required'```

Reply:
I did that because only the key is needed and the value is optional